src/Altinn.Platform/Altinn.Platform.Events/Events/Services/SubscriptionService.cs

```diff
@@ -51,7 +51,7 @@ public async Task<List<Subscription>> GetOrgSubscriptions(string source, string
         {
             List<Subscription> searchresult = await _repository.GetSubscriptionsByConsumer("/org/%");
             return searchresult.Where(s =>
-                s.SourceFilter.Equals(source) &&
+                source.StartsWith(s.SourceFilter.OriginalString) &&
```

Comment:
I'm not familiar with the data being passed in here or the database, but is the source parameter supposed to be verified against the original string or should it be the other way around? That you check if the original string starts with the source parameter?

Reply:
@RonnyB71 the source would typical be something like this https://skd.apps.altinn.no/skd/skattemelding/instances/234234422/2acb1253-07b3-4463-9ff5-60dc82fd59f8

The subscription source would be something like this https://skd.apps.altinn.no/skd/skattemelding/